flink-runtime/src/main/java/org/apache/flink/runtime/state/AbstractKeyedStateBackend.java

```diff
@@ -424,14 +424,6 @@ public <N, S extends State> S getPartitionedState(
             return (S) lastState;
         }
 
-        InternalKvState<K, ?, ?> previous = keyValueStatesByName.get(stateDescriptor.getName());
```

Comment:
Please could you explain in the Jira or PR why we can remove this code please.

Reply:
Updated. Thank you.